docs/en/loading/loading_introduction/feature-support-loading-and-unloading.md

```diff
@@ -0,0 +1,627 @@
+
+[2]\: When the security protocol is set to `sasl_plaintext` or `sasl_ssl`, both SASL and GSSAPI (Kerberos) authentications are supported.
+
+[3]\: When the security protocol is set to `sasl_plaintext` or `sasl_ssl`, both SASL and PLAIN authentications are supported.
+
+:::
+
+### Unloading - Authentication
+
+|                 | INSERT INTO FILES | EXPORT |
+| :-------------- | :----------------: | :-----: |
+| Single Kerberos | /                 | /      |
+
+## Loading - Other parameters and features
+
+<table align="center">
+    <tr>
+        <th rowspan="2">Parameter and feature</th>
+        <th colspan="8">Loading method</th>
+    </tr>
+    <tr>
+        <th>Stream Load</th>
+        <th>INSERT from FILES</th>
+        <th>INSERT from SELECT/VALUES</th>
+        <th>Broker Load</th>
+        <th>PIPE</th>
+        <th>Routine Load</th>
+        <th>Spark Load</th>
+    </tr>
+    <tr>
+        <td>partial_update</td>
+        <td>Yes (v3.0+)</td>
+        <td colspan="2">Yes [1] (v3.3+)</td>
+        <td>Yes (v3.0+)</td>
+        <td>-</td>
+        <td>Yes (v3.0+)</td>
+        <td>/</td>
+    </tr>
+    <tr>
+        <td>partial_update_mode</td>
+        <td>Yes (v3.1+)</td>
+        <td colspan="2">/</td>
+        <td>Yes (v3.1+)</td>
+        <td>-</td>
+        <td>/</td>
+        <td>/</td>
+    </tr>
+    <tr>
+        <td>COLUMNS FROM PATH</td>
+        <td>-</td>
+        <td>Yes (v3.2+)</td>
+        <td>-</td>
+        <td>Yes</td>
+        <td>-</td>
+        <td>-</td>
+        <td>Yes</td>
+    </tr>
+    <tr>
+        <td>timezone or session variable time_zone [2]</td>
+        <td>Yes [3]</td>
+        <td>Yes [4]</td>
+        <td>Yes [4]</td>
+        <td>Yes [4]</td>
+        <td>/</td>
+        <td>Yes [4]</td>
+        <td>/</td>
+    </tr>
+    <tr>
+        <td>Time accuracy - Microsecond</td>
+        <td>Yes</td>
+        <td>Yes</td>
+        <td>Yes</td>
+        <td>Yes (v3.1.11+/v3.2.6+)</td>
+        <td>/</td>
+        <td>Yes</td>
+        <td>Yes</td>
+    </tr>
+</table>
+
+:::note
+
+[1]\: From v3.3 onwards, StarRocks supports Partial Updates in Row mode for INSERT INTO by specifying the column list.
+
+[2]\: Setting the time zone by the parameter or the session variable will affect the results returned by functions such as strftime(), alignment_timestamp(), and from_unixtime().
+
+[3]\: Only the parameter `timezone` is supported.
+
+[4]\: Only the session variable `time_zone` is supported.
+
+:::
+
+## Unloading - Other parameters and features
+
+<table align="center">
+    <tr>
+        <th>Parameter and feature</th>
+        <th>INSERT INTO FILES</th>
+        <th>EXPORT</th>
+    </tr>
+    <tr>
+        <td>target_max_file_size</td>
+        <td rowspan="3">Yes (v3.2+)</td>
+        <td rowspan="4">/</td>
+    </tr>
+    <tr>
+        <td>single</td>
+    </tr>
+    <tr>
+        <td>Partitioned_by</td>
+    </tr>
+    <tr>
+        <td>Session variable time_zone</td>
```

Comment:
modification  for Session variable time_zone & Time accuracy - Microsecond for unloading
It's supported in 3.3.0

Reply:
But maybe not tested well